ProcessLib/LIE/SmallDeformation/Tests.cmake

```diff
@@ -89,10 +89,14 @@ AddTest(
     TESTER vtkdiff
     REQUIREMENTS NOT OGS_USE_MPI
     DIFF_DATA
-    GLOB cohesive_zone_load_path_pcs_0_ts_*.vtu displacement displacement 1e-16 0
```

Comment:
Put back the comparison of displacement, displacement_jump1, and sigma_*.
Use GLOB pattern, instead of writing out each time step.
The tolerances for the stress_n are rather loose; how come?

Reply:
The GLOB pattern sounds like making the file comparison with an identical file name in different folder. I am not sure.

Reply:
Yes, it does compare the files from the source folder (expected output) to the build folder (ctest output) and the files are named identically. Simply do as for displacement etc.

Reply:
The expected output contains the analytical solution 'f_stress_n_analytical' which is not included in the ctest output though...